package-lock.json

```diff
@@ -1007,11 +1007,6 @@
       "resolved": "https://registry.npmjs.org/is/-/is-3.3.0.tgz",
       "integrity": "sha512-nW24QBoPcFGGHJGUwnfpI7Yc5CdqWNdsyHQszVE/z2pKHXzh7FZ5GWhJqSyaQ9wMkQnsTx+kAI8bHlCX4tKdbg=="
     },
-    "is-absolute-url": {
```

Comment:
Slightly concerned that we are removing packages here

Reply:
This is a fresh `npm install` on macOS Big Sir and everything worked fine.

Reply:
I'd suggest we just go with this and if there's a problem, it's very straightforward to back out.

Reply:
Sounds good